layers/README.md

```diff
@@ -18,6 +18,13 @@ This folder contains modules and scripts for working with ATT&CK Navigator layer
 |:-------|:------------|
 | [layerops](manipulators/layerops.py) | Provides a means by which to combine multiple ATT&CK layer objects in customized ways. A further breakdown can be found in the corresponding section below. |
 
+#### Exporter Scripts
+| script | description |
+|:-------|:------------|
+| [matrix_gen](exporters/matrix_gen.py) | Provides a means by which to generate a matrix from raw data, either cached or from the ATT&CK TAXII server. |
+| [excel_templates](exporters/excel_templates.py) | Provides a means by which to convert a matrix into a base excel template. |
```

Comment:
Do these two even need to get mentioned? Users of the library probably won't ever need to access matrix_gen or excel_templates, those are really just utilities for the development of to_excel (and possibly other exporters later).

Reply:
While they aren't likely to be used, I still feel that mentioning them is the more user-friendly approach, since there are functions that can be used for user tasks.

Reply:
OK. I think we might want to move them into their own section, like "helper functions" or something with a description saying something to the effect of "for advanced users."

Reply:
I've put them as 'utility functions'. Let me know if you think more elaboration is needed.